wiki/History_of_osu!/Online_rankings/en.md

```diff
@@ -18,19 +18,19 @@ Numerous players have held rank #1 in each game mode under different ranking sys
 
 *Main page: [osu!](osu!)*
 
-As of July 19, 2021, on the [osu! performance rankings](https://osu.ppy.sh/rankings/osu/performance) the rank #1 player is ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902/osu).
+As of July 21, 2024, on the [osu! performance rankings](https://osu.ppy.sh/rankings/osu/performance) the rank #1 player is ::{ flag=GB }:: [9MlCE](https://osu.ppy.sh/users/9269034/osu).
 
 ## ![][osu!taiko] osu!taiko
 
 <!-- TODO: Add ranking history page similar to osu! -->
 
-As of October 9, 2023, on the [osu!taiko performance rankings](https://osu.ppy.sh/rankings/taiko/performance) the rank #1 player is ::{ flag=JP }:: [Majimanjiwwwww](https://osu.ppy.sh/users/12337864/taiko).
+As of June 6, 2024, on the [osu!taiko performance rankings](https://osu.ppy.sh/rankings/taiko/performance) the rank #1 player is ::{ flag=JP }:: [shinchikuhome](https://osu.ppy.sh/users/3174184/taiko).
```

Comment:
src for this date? (dont need to include in the article i just don't know how to check this)

Reply:
Seems like the #11727 had the wrong date. 
According to [this reddit post](https://www.reddit.com/r/osugame/comments/1djjahk/taiko_majimanjiwwwwws_account_disappears_making/), a [tweet by shinchikuhome](https://x.com/shinchikuhome/status/1803338034567385378) and 
[shinchikuhome's user profile graph](https://osu.ppy.sh/users/3174184/taiko) all point to them retaking rank 1 on `June 19, 2024`.  
```suggestion
As of June 19, 2024, on the [osu!taiko performance rankings](https://osu.ppy.sh/rankings/taiko/performance) the rank #1 player is ::{ flag=JP }:: [shinchikuhome](https://osu.ppy.sh/users/3174184/taiko).
```